### DISCUSSION TITLE
How to set `visit` number priority

### DISCUSSION THREAD

**Discussion by chookity-pokk** [meow-edit/meow#427] [Q&A]
Sorry for the confusing title, I'm not exactly sure how to phrase it. When I search with visit it appears after results from [blamer](https://github.com/Artawower/blamer.el), and makes my numbers of possible visits appears off screen. I got a [screenshot](https://imgur.com/a/vhxCH1f) of it to hopefully make this more clear. If you go to the right of the highlighted line you can see the results from blamer and then the first number for visit. I was wondering how I can change the visit priority to hopefully have the appear first?

**Comment by Linerre** [comment]
> makes my numbers of possible visits appears off screen.

To my eyes, this seems to be more about your config for line wrapping in a buffer.

My understanding is that `visit` just searches for target text in the current buffer (overlay included), highlights all matches, and jumps to the one nearest your current point.

A quick look at the source code, it could be that `meow-visit` calls `setq` to always make the internal var `eow--dont-remove-overlay` to `true`:
https://github.com/meow-edit/meow/blob/d80c2ff2f94333492dd7714c4210440162985b20/meow-command.el#L1376 

I'm not sure if this variable could be made into a user-definable one. Thus in your case, you could disable the number, rather than having them displayed in a priority you prefer.

**Reply by DogLooksGood** [reply]
It's probably unrelated to the question. It was asking how to put the `[index/count]` overlay of visit count in front of the overlay made by blamer.

Unfortunately, I don't have an answer for now.  Have to do some research first.
